dev/ComboBox/ComboBox_themeresources.xaml

```diff
@@ -604,7 +611,7 @@
                         </Grid.RowDefinitions>
                         <Grid.ColumnDefinitions>
                             <ColumnDefinition Width="*" />
-                            <ColumnDefinition Width="32" />
+                            <ColumnDefinition Width="38" />
```

Comment:
Shouldn't it be 40? -4 margin was changed to 4 so it give +8 size difference?

Reply:
Yes, but it was actually mispositioned by 2 pixels before this change, which there is another internal issue for.